### DISCUSSION TITLE
No video detected in Version 9 🙏  please upgrade to v9.0.1.18

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#538] [Announcements]
If you use an old version of Chrome or Edge, it's possible that you won't see any videos.

I broke that in version 9 … sorry 🙏

We fixed this issue in version 9.0.1.18, it's coming soon (waiting for Google to update everybody).

**Comment by SaturnSighed** [comment]
It's pretty irritating. I paid for the license, got suckered by the promise of a wonderful upgrade and now I have nothing. I've uninstalled the extension and wish there was an easy way to warn everyone DON'T UPGRADE

**Reply by paulrouget** [reply]
Sorry for the bug. It's only affecting a small portion of our users. We're waiting for Google to release the fix.

**Reply by Gablitzer** [reply]
me to, Firefox and download helper was ok for years, after new version update and purchasing the licence (verified!) the icon is shadowed and no more operative.

**Comment by paulrouget** [comment]
This is now fixed. Please upgrade to version 9.0.1.18 on Chrome & Edge.

**Reply by sorats515** [reply]
since 06-09-24, it doesn't work any more, on any site your are going and I paid for a licence. I've already download the new version several times. What can I do please?

**Reply by sorats515** [reply]
just got the last version of Chrome and the last version of VDHelper. Nothing works

**Reply by gridsleep** [reply]
If you are using VPN, try disabling it or changing the location.

**Reply by sorats515** [reply]
don't have any VPN

**Reply by sorats515** [reply]
it works now, I have uninstalled Ublock

**Reply by gridsleep** [reply]
Or, you could have added VDH to the list of trusted sites in uBlock.. Now, whatever you were blocking with uBlock is now less blocked.

**Comment by GrampaWildWilly** [comment]
Seems like as appropriate a place as any to post this.  This is with Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/00fda153-a6d2-4792-9173-83e1edce5f95)

Sorry, can't share the URL.  You know I would normally, but not this time.  The subwindow listing all the resolutions is cut off here.  The not visible part up higher is supposed to show a 1080p & a 2160p entry.  I was particularly interested in the bit rates on those entries.

By removing the first 2 entries on the VDH menu, partially obscured in that image above, I was able to get this:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f463d6cb-3d5e-49d5-a236-336c6b3eda18)

But that really shouldn't have been necessary.

**Reply by paulrouget** [reply]
I don't have a fix yet, but it's a known issue.

**Comment by kiwiarthur** [comment]
Upgraded both Chrome and VDH to latest versions (9.0.1.18). Still doesn't work for me.

**Reply by kiwiarthur** [reply]
Have now update to Chrome Version 125.0.6422.113 (Official Build) (64-bit) with VDH 9.0.2.3 - still not working and now Firefox with 9.0.2.3 no longer working.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

You really ought to open separate discussions to pursue problem diagnosis & resolution.

**Comment by lonetlove** [comment]
Upgraded to Version 9.0.2.3. No video detected. Still bugged?

**Comment by gridsleep** [comment]
VDH 9.0.2.3 is hardly working in Firefox in Windows 10 x64. Once, the drop down window said VDH would not work in a private window unless it is enabled in settings, only this was not in a private window. Enabling that didn't make any difference. I think the 'tube has found a new way to block VDH. It works still with some videos but more and more it simply does not see any downloadable content.

UPDATE: Never mind. It was AVG VPN causing the disconnect. That seems puzzling since the VPN sits between my computer and the ISP server, not between VDH and Youtube which is probably a totally different route. The video is not being scavenged through my computer, so what would my VPN have to do with VDH's connection? I assumed that my request for a download was just instructions going to VDH server and that connection having nothing to do with my link to the 'tube. I've worked in IT for decades and my understanding of the 'net can't be that wrong, can it?
Also, the only problem is through servers labeled New York City. There is a peculiar one in AVG VPN called Gotham which also goes through New York City but ostensibly is named not to look as if it is New York City. That one does not interfere with VDH, but it keeps my Thunderbird email program from connecting with my ISP email server. I have yet to find a VPN server that works in all instances with all connections without problem. Everybody out there has a problem with something and all it does is give me a headache.

**Reply by Mattmjs** [reply]
I don't think there is a VDH server, all downloads take place on your own machine. Problems can occur if you use a browser
based vpn especially if it requires the companion app for the download. Because browser based vpns only encrypt
browser traffic. The companion app is an app that works on the computer and browser vpns don't encrypt any activity on
the system. You want to a use system level desktop vpn connection for downloading with VDH. Is your vpn free or paid ?
Paid is always better than free.

**Reply by gridsleep** [reply]
I use AVG VPN, not a browser VPN. Browser VPN is pointless because it leaves the rest of the computer open and only obscures HTTP traffic. That is not security. As I wrote, I switched to a different VPN district (in the same city) and VDH works again. Only now my Thunderbird mail is being blocked. I will have to go back to viewing my email in the browser. Nothing works perfectly in this technological world that has its hands around our throats, and no one seems to be alarmed by that.
I think there is a VDH server at least for routing. Attempts to download videos to my computer are blocked at the source. There has to be an intermediary server that is declaring itself to be a relay or pass through in order to get unfiltered access to 'tube videos. VDH is doing something my computer cannot do. There is no other way this could work.

**Comment by JJ-Luc** [comment]
Mint -Ubuntu SMP + Firefox 126.0.1 (64bits) + vdh 19.1 & coapp  .
No way to download : even if the video appear in the list, vdh fails to download or downloads only a video of a few seconds. 
If it downloads (a few seconds), it place it automatically allways in thge same place. In the previews ersion, i could choice the way and the PLACE the video will be placed. Automatic choice is not allways the best ! I prefer dispatch in different directories.
Is it possible to reverse this upgrade to a previews version ?

**Comment by M0r1en** [comment]
Using licensed version 2.0.2.3 and vdhcoapp.exe v2.0.19. Did not detect any video.
Disabled VPN, as mentioned before, and all works fine! 
It would be better if you could download with VPN enabled as before the update. Why is it changed?

**Reply by gridsleep** [reply]
Software cannot account for data blocking on any particular server. It depends on who set up that server and there are millions of them. It's actually a wonder any of it works at all. Try rerouting through a different location. Sometimes rerouting to the same place gets a different server there. Sometimes it is the VPN software and the servers they employ. I am currently testing Mozilla VPN after using AVG VPN for years. The AVG has some problems connecting some applications through some servers, and I have to keep switching locations to get one or another thing to connect. Mozilla, so far, seems to connect everything without a blockage. Your mileage may vary. Also, remember that Google are constantly changing their machinery to try to block anything they don't authorize, even though it's a fine balancing act between protecting their investment and annoying their customers.

**Comment by digitaljoeB** [comment]
NOT WORKING.
Needs to be fixed now or refund everyone please

**Reply by jcfrog** [reply]
Sorry for this, there is a new issue with youtube but you can try this:  https://github.com/aclap-dev/video-downloadhelper/discussions/1074#discussioncomment-11914403

**Comment by digitaljoeB** [comment]
It worked!!!

Joe Barnes
Associate Clinical Professor of Marketing, Albers School of Business &
Economics, Seattle University.
Author:* "**Social Media Safety: What Parents and Teens Need to Know"*
<http://goog_342832726/>https://shorturl.at/VdqQ5
http://www.linkedin.com/in/joebarnesseattle
*"Be Kind. Be Compassionate. Have Humility."*

**Reply by ITProfeska** [reply]
https://github.com/aclap-dev/video-downloadhelper/discussions/2741
